### DISCUSSION TITLE
Institutional email verification

### DISCUSSION THREAD

**Discussion by steven0seagal** [RosettaCommons/rosetta#158] [ROSIE] [answered]
Hello,
I'd like to access ROSIE-2 but I'm getting an email verification error. My facility is Institute of Biochemistry and Biophysics
Polish Academy of Sciences https://ibb.edu.pl/en/ and mail ending is @ibb.waw.pl
Best

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! This should be fixed now, please feel free to re-open if you still having issues with login.
